glib/src/object.rs

```diff
@@ -3231,6 +3248,16 @@ impl<T: ObjectType> ObjectExt for T {
         }
     }
 
+    fn add_weak_ref_notify<F: FnOnce() + Send + Sync + 'static>(&self, f: F) -> WeakRefNotify<T> {
```

Comment:
```suggestion
    fn add_weak_ref_notify<F: FnOnce() + Send + 'static>(&self, f: F) -> WeakRefNotify<T> {
```

Is actually enough here. It's an `FnOnce`, it's not accessed from multiple threads at the same time